cmd/kubeadm/app/cmd/phases/reset/updateclusterstatus.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"errors"
+	"os"
+
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	"k8s.io/kubernetes/cmd/kubeadm/app/phases/uploadconfig"
+)
+
+// NewUpdateClusterStatus creates a kubeadm workflow phase for update-cluster-status
+func NewUpdateClusterStatus() workflow.Phase {
+	return workflow.Phase{
+		Name:  "update-cluster-status",
+		Short: "Remove control-plane from cluster status.",
+		Long:  "Remove control-plane from cluster status.",
```

Comment:
nit on wording maybe?
```
Short: "update-cluster-status for removed control-plane node",
Long: "update-cluster-status removes the control plane node from the cluster status and uploads to the cluster."
```
idk. just think it would be nice if the short and long where different. maybe it's not a big deal.

Reply:
Thanks, I'll update it in a bit